main.go

```diff
@@ -18,38 +18,24 @@ package main
 
 import (
 	"flag"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/client/meta"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/logger"
+	"github.com/kiegroup/kogito-cloud-operator/core/client"
+	"github.com/kiegroup/kogito-cloud-operator/core/logger"
+	"github.com/kiegroup/kogito-cloud-operator/meta"
 	"os"
 
-	"k8s.io/apimachinery/pkg/runtime"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	appv1beta1 "github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
 	"github.com/kiegroup/kogito-cloud-operator/controllers"
 	// +kubebuilder:scaffold:imports
 )
 
 var (
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme   = meta.GetRegisteredSchema()
+	setupLog = logger.GetLogger("setup")
 )
 
-func init() {
-	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
-
-	utilruntime.Must(appv1beta1.AddToScheme(scheme))
-	// all schemas required by Kogito Operator
-	builder := meta.GetRegisteredSchemeBuilder()
-	utilruntime.Must(builder.AddToScheme(scheme))
-	// +kubebuilder:scaffold:scheme
-}
```

Comment:
I think we shouldn't remove this function and change the global scheme var. Adding to the `init` function is the recommended way: https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#adding-3rd-party-resources-to-your-operator.

Plus, the `// +kubebuilder:scaffold:scheme` annotation is important to keep.

Reply:
`// +kubebuilder:scaffold:scheme` is already commented. I didn't make any change into it.

I removed `init` function because `scheme   = meta.GetRegisteredSchema()` doing same thing as we performed in `init` function. It will add `init` back and initialize `scheme` through it.

```
var (
	scheme   = runtime.NewScheme()
	setupLog = ctrl.Log.WithName("setup")
)

func init() {
	utilruntime.Must(clientgoscheme.AddToScheme(scheme))

	utilruntime.Must(appv1beta1.AddToScheme(scheme))
	// all schemas required by Kogito Operator
	builder := meta.GetRegisteredSchemeBuilder()
	utilruntime.Must(builder.AddToScheme(scheme))
	// +kubebuilder:scaffold:scheme
}
```

Reply:
Thanks! But the annotation must be kept. It's commented, but it's used by kubebuilder to scaffold the scheme. Can you, please, put it back?

Reply:
I didn't remove commented annotation.
https://github.com/vaibhavjainwiz/kogito-cloud-operator/blob/6d84bd5313d272b2729256c7bc292df9d0602153/main.go#L32